textual_fspicker/base_dialog.py

```diff
@@ -70,8 +72,10 @@ class FileSystemPickerScreen(ModalScreen[Optional[Path]]):
     BINDINGS = [Binding("full_stop", "hidden"), Binding("escape", "dismiss(None)")]
     """The bindings for the dialog."""
 
+    select_Changed = False
```

Comment:
Couple of things here:

1. It's not clear to me why this needs to be a class variable.
2. Variables within this code should always follow the "usual" Python convention; so this would be `select_changed` (all lower-case).

Reply:
this varible is just a way to prevent the select init from triggering the DirectoryNavigation.location.setter twice in a row first with the given path and then from finding mounted disks.

But I know this is not a good solution.